tutorials/plot_brainstorm_auditory.py

```diff
@@ -0,0 +1,362 @@
+Brainstorm auditory tutorial dataset
+====================================
+
+Here we compute the evoked from raw for the auditory Brainstorm
+tutorial dataset. For comparison, see [1]_ and
+http://neuroimage.usc.edu/brainstorm/Tutorials/Auditory
+
+Experiment:
+    - One subject 2 acquisition runs 6 minutes each.
+    - Each run contains 200 regular beeps and 40 easy deviant beeps.
+    - Random ISI: between 0.7s and 1.7s seconds, uniformly distributed.
+    - Button pressed when detecting a deviant with the right index finger.
+
+The specifications of this dataset were discussed initially on the FieldTrip
+bug tracker:
+http://bugzilla.fcdonders.nl/show_bug.cgi?id=2300
+
+References
+----------
+.. [1] Tadel F, Baillet S, Mosher JC, Pantazis D, Leahy RM.
+       Brainstorm: A User-Friendly Application for MEG/EEG Analysis.
+       Computational Intelligence and Neuroscience, vol. 2011, Article ID
+       879716, 13 pages, 2011. doi:10.1155/2011/879716
+"""
+
+# Authors: Mainak Jas <mainak.jas@telecom-paristech.fr>
+#          Eric Larson <larson.eric.d@gmail.com>
+#          Jaakko Leppakangas <jaeilepp@student.jyu.fi>
+#
+# License: BSD (3-clause)
+
+import os.path as op
+import pandas as pd
+import numpy as np
+
+import mne
+from mne import combine_evoked
+from mne.minimum_norm import apply_inverse
+from mne.datasets.brainstorm import bst_auditory
+from mne.io import read_raw_ctf
+from mne.filter import notch_filter, low_pass_filter
+
+print(__doc__)
+
+###############################################################################
+# To reduce memory consumption and running time, some of the steps are
+# precomputed. To run everything from scratch change this to False. With
+# ``use_precomputed = False`` running time of this script can be several
+# minutes even on a fast computer.
+use_precomputed = True
+
+###############################################################################
+# The data was collected with a CTF 275 system at 2400 Hz and low-pass
+# filtered at 600 Hz. Here the data and empty room data files are read to
+# construct instances of :class:`mne.io.Raw`.
+data_path = bst_auditory.data_path()
+
+subject = 'bst_auditory'
+subjects_dir = op.join(data_path, 'subjects')
+
+raw_fname1 = op.join(data_path, 'MEG', 'bst_auditory',
+                     'S01_AEF_20131218_01.ds')
+raw_fname2 = op.join(data_path, 'MEG', 'bst_auditory',
+                     'S01_AEF_20131218_02.ds')
+erm_fname = op.join(data_path, 'MEG', 'bst_auditory',
+                    'S01_Noise_20131218_01.ds')
+
+###############################################################################
+# In the memory saving mode we use ``preload=False`` and use the memory
+# efficient IO which loads the data on demand. However, filtering and some
+# other functions require the data to be preloaded in the memory.
+preload = not use_precomputed
+raw = read_raw_ctf(raw_fname1, preload=preload)
+mne.io.concatenate_raws([raw, read_raw_ctf(raw_fname2, preload=preload)])
```

Comment:
raw = read_raw_ctf([raw_fname1, raw_fname2], preload=preload)

works? if so I think it's simpler